pkg/drivers/jetstream/jetstream.go

```diff
@@ -0,0 +1,900 @@
+package jetstream
+
+import (
+	"context"
+	"encoding/json"
+	"regexp"
+	"sort"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/k3s-io/kine/pkg/drivers/jetstream/kv"
+	"github.com/k3s-io/kine/pkg/server"
+	"github.com/nats-io/nats.go"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	kineBucket             = "kine"
+	revHistory             = 12
+	slowMethodMilliseconds = 500
+)
+
+var (
+	toplevelKeyMatch = regexp.MustCompile(`/registry/(.*)/.*`)
```

Comment:
K3s stores cluster bootstrap data outside the Kubernetes keyspace under the `/bootstrap/` prefix - does this need any special consideration?
https://github.com/k3s-io/k3s/blob/04c39ba621d1ca30086a96694d74fe23f2337228/pkg/cluster/encrypt.go#L21-L23

Reply:
Ah yes... I should probably just add handling for it just in case - I don't think as written any problems would occur but it won't hurt to handle that `/bootstrap/` case

Reply:
Modified regex to allow for other TL folders like `/bootstrap`